source/tutorials/installing-nixos-on-a-raspberry-pi.md

```diff
@@ -42,6 +42,8 @@ $ nix-shell -p wget zstd
 :::{note}
 You can pick a newer image by going to [Hydra job](https://hydra.nixos.org/job/nixos/trunk-combined/nixos.sd_image.aarch64-linux),
 clicking on a build and copying the link to the build product image.
+There have been reported [some problems](https://github.com/NixOS/nixpkgs/issues/179701) with the latest sd images on the pi 4, where it turns of the display while booting up.
+If you are experiencing such problems, please use [this image](https://hydra.nixos.org/build/134720986/download/1/nixos-sd-image-21.03pre262561.581232454fd-aarch64-linux.img.zst) instead until it is fixed in the newest sd images.
```

Comment:
```suggestion
If you are experiencing such problems, please use [the most recent working build](https://hydra.nixos.org/build/134720986/download/1/nixos-sd-image-21.03pre262561.581232454fd-aarch64-linux.img.zst) instead until the issue is fixed.
```

Name links in a way that is self-explanatory. The link should be readable in isolation. 

Also in general, don't just point people to "this random-sounding thing", that doesn't exactly inspire confidence and requires additional scrutiny. Make evident why this is the correct approach to follow. In this case it's only a matter of telling what it is you're pointing to.